File: testLib/src/main/java/me/ycdev/android/lib/test/base/NormalJUnitBase.java
package me.ycdev.android.lib.test.base;

import org.junit.BeforeClass;

import me.ycdev.android.lib.test.log.TimberJvmTree;
import timber.log.Timber;

public class NormalJUnitBase {
    @BeforeClass
    public static void setupClass() {
        Timber.plant(new TimberJvmTree());
    }
}

File: testLib/src/main/java/me/ycdev/android/lib/test/base/NormalJUnitBase.kt
package me.ycdev.android.lib.test.base

import me.ycdev.android.lib.test.log.TimberJvmTree
import org.junit.BeforeClass
import timber.log.Timber

open class NormalJUnitBase {
    companion object {
        @BeforeClass @JvmStatic
        fun setupClass() {
            Timber.plant(TimberJvmTree())
        }
    }
}
